pkg/reconciler/common/prune.go

```diff
@@ -120,7 +124,7 @@ func Prune(ctx context.Context, k kubernetes.Interface, tC *v1alpha1.TektonConfi
 	if pruningNamespaces.commonScheduleNs != nil && len(pruningNamespaces.commonScheduleNs) > 0 {
 		jobs := pruner.createAllJobContainers(pruningNamespaces.commonScheduleNs)
 		cronNameWithRandomSuffix := SimpleNameGenerator.RestrictLengthWithRandomSuffix(CronName)
-		if err := pruner.checkAndCreate(ctx, cronNameWithRandomSuffix, tC.Spec.TargetNamespace, tC.Spec.TargetNamespace, tC.Spec.Pruner.Schedule, jobs, ownerRef, pruneCronLabel); err != nil {
+		if err := pruner.checkAndCreate(ctx, cronNameWithRandomSuffix, tC.Spec.TargetNamespace, "", tC.Spec.Pruner.Schedule, jobs, ownerRef, pruneCronLabel, pruningNamespaces.configChanged); err != nil {
```

Comment:
`pruningNamespaces.configChanged`will this ever become `true`in this call.

how does `pruningNamespaces, err := prunableNamespaces(ctx, k, tC.Spec.Pruner)`change `configChanged`